src/pages/dashboard/profile/index.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { Col, Row } from "reactstrap";
+import AuthInput from "../../../components/inputs/auth-input";
+import DetailView from "../../../components/views/detail-view";
+import { useChangePassword } from "../../../hooks/useChangePassword";
+
+const ChangePassword = () => {
```

Comment:
Good work here. Please change the component name to Profile
```suggestion
const Profile = () => {
```